### ISSUE TITLE
[Bug]: Localhost Proxy usage breaks local invidious instance

### ISSUE THREAD

**Issue by fugbixer** [FreeTubeApp/FreeTube#2339]
### Proxy usage combined with local instance does not work and result in broken local instance

Tested with http proxy (unknown about sock4,socks5 & https )

I use a **localhost http proxy** which is a fail-safe VPN redirect

https://github.com/binhex/arch-delugevpn

**Error: Status code: 502**

**Proxy usage combined with local instance does not work and result in broken local instance**
using website instance from someone else works fine, sadly does lead to more traffic on their end.

### Expected Behavior

Either use proxychain or a similar tool to redirect traffic through the proxy or notice the user that for local instance the proxy setting will be ignored and than you ignore the proxy for local instance requests.

### Issue Labels

accessibility issue, API issue, content not loading, usability issue

### FreeTube Version

v0.16.0 Beta

### Operating System Version

Arch Aur Package

**Comment by absidue** [comment]
Please check that you have entered the URL for your invidious instance correctly so that it's a valid URL, don't forget the `http://` at the start of it. If that doesn't help then the issue is with your invidious instance or your proxy.

**Comment by absidue** [comment]
As you've edited out the relevant form field about which API you are using, have you set FreeTube to "Local API" or "Invidious API"?

**Comment by fugbixer** [comment]
Instance: https://yewtu.be

Im using the web instance currently because with proxy it does not work.

When i use local API and disable Fallback to Non-preferred 

Error: Status code: 502 (Clicked copy)


Local API Error (Click to copy): Error: Status code: 502 (Copied manual)


It cant load the video, the metadata of the video but the commend query does work.
The proxy does work well but my guess is in case that the issue is not related to all proxies.
Can it be the based on the fact that my proxy uses localhost (privoxy) but that should also lead to issues with usage of tor in combination of local API.

Video tested on: https://youtu.be/FccdqCucVSI

But also when i test the proxy settings nomatter what if i use privoxy or tor i get in freetube.

> Error getting network information. Is your proxy configured properly? 

Okay, i tested both types. Socks5 with tor and http with privoxy both can use the local API and work with the web api. Both produce test error message within freetubes proxy settings test feature!

Videos loaded over local API : 

```

Play Video

SUBSCRIBE
Published on Jan 1, 1970
0 views

Top comments
Sort by
Comments Hide Comments

 Pinned by

The Linux Experiment 10 Hours ago

Download Safing's Portmaster, or subscribe to the SPN, and take control of your network traffic: [safing.io/portmaster](https://safing.io/portmaster)

 2

Arnie's Tech 1 Hour ago

Gonna make a video on this surely. So few people can imagine how much of an impact non-code or even non-financial contribution can have! 💪🙏

 20  View 1 reply


Michael Placzek 1 Hour ago

Just using FOSS projects and talking about its pros and cons is very helpful

 24  View 5 replies
```

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@fugbixer do you still face this issue?
